chemprop/featurizers/atom.py

```diff
@@ -31,54 +38,50 @@ class MultiHotAtomFeaturizer(AtomFeaturizer):
-    | 108-114             | formal charge   | Y            |
-    +---------------------+-----------------+--------------+
-    | 114-119             | chiral tag      | Y            |
-    +---------------------+-----------------+--------------+
-    | 119-125             | # Hs            | Y            |
-    +---------------------+-----------------+--------------+
-    | 125-131             | hybridization   | Y            |
-    +---------------------+-----------------+--------------+
-    | 131-132             | aromatic?       | N            |
-    +---------------------+-----------------+--------------+
-    | 132-133             | mass            | N            |
-    +---------------------+-----------------+--------------+
-
-    NOTE: the above signature only applies for the default arguments, as the each slice (save for
-    the final two) can increase in size depending on the input arguments.
+    **NOTE**: All the features, except for aromaticity and mass, are padded with an 0 for unknown values.
+
+    Parameters
+    ----------
+    atomic_nums : Sequence[int], default=list(range(1, 37)) + [53]
+        type of atom (ex. C, N, O), by atomic number
+    degree : Sequence[int], default=list(range(6))
+        number of bonds the atom is involved in
+    formal_charges : Sequence[int], default=[-1, -2, 1, 2, 0]
+        integer electronic charge assigned to atom
+    chiral_tags : Sequence[int], default=list(range(4))
+        unspecified, tetrahedral CW/CCW, or other
+    num_Hs : Sequence[int], default=list(range(5))
+        number of bonded hydrogen atoms
+    hybridizations : Sequence[int], default=[HybridizationType.S,
+    HybridizationType.SP, HybridizationType.SP2, HybridizationType.SP2D,
+    HybridizationType.SP3, HybridizationType.SP3D, HybridizationType.SP3D2]
+        type of atom’s hybridization (ex. sp, sp2, sp3, sp3d, or sp3d2)
     """
 
-    max_atomic_num: InitVar[int] = 100
-    degrees: Sequence[int] = field(default_factory=lambda: range(6))
-    formal_charges: Sequence[int] = field(default_factory=lambda: [-1, -2, 1, 2, 0])
-    chiral_tags: Sequence[int] = field(default_factory=lambda: range(4))
-    num_Hs: Sequence[int] = field(default_factory=lambda: range(5))
-    hybridizations: Sequence[HybridizationType] = field(
-        default_factory=lambda: [
+    def __init__(
+        self,
+        atomic_nums: Sequence[int] = list(range(1, 37)) + [53],
+        degrees: Sequence[int] = list(range(6)),
+        formal_charges: Sequence[int] = [-1, -2, 1, 2, 0],
+        chiral_tags: Sequence[int] = list(range(4)),
+        num_Hs: Sequence[int] = list(range(5)),
+        hybridizations: Sequence[int] = [
+            HybridizationType.S,
             HybridizationType.SP,
             HybridizationType.SP2,
+            HybridizationType.SP2D,
             HybridizationType.SP3,
             HybridizationType.SP3D,
             HybridizationType.SP3D2,
-        ]
```

Comment:
you shouldn't use lists (or mutable objects in general) as default arguments to functions.